docs/src/appendix/cheatcodes.md

```diff
@@ -6,3 +6,12 @@
 * [`stop_roll`](cheatcodes/stop_roll.md)
 * [`start_warp`](cheatcodes/start_warp.md)
 * [`stop_warp`](cheatcodes/stop_warp.md)
+
+> ℹ️ **Info**
+> To use cheatcodes you need to add `snforge_std` package as a dependency in
+> your [`Scarb.toml`](https://docs.swmansion.com/scarb/docs/guides/dependencies#adding-a-dependency) 
```

Comment:
@mkaput made the [PR](https://github.com/foundry-rs/starknet-foundry/pull/428) which updates scarb links (it is explained inside). We should use this link instead https://docs.swmansion.com/scarb/docs/guides/dependencies.html#adding-a-dependency

Reply:
And please check all the new ones 🙏

Reply:
Sure, will do